Lib/datetime.py

```diff
@@ -925,7 +925,33 @@ def ctime(self):
             self._day, self._year)
 
     def strftime(self, fmt):
-        "Format using strftime()."
+        """strftime(fmt) -> date_string
+
+        Return a string from the date parsed according to format.
+        Format codes referring to hours, minutes or seconds will see 0 values.
+
+        Args:
+            format: representation of date_string using format codes.
+
+            Commonly used format codes:
+                %Y  Year with century as a decimal number.
+                %m  Month as a decimal number [01,12].
+                %d  Day of the month as a decimal number [01,31].
+                %H  Hour (24-hour clock) as a decimal number [00,23].
+                %M  Minute as a decimal number [00,59].
+                %S  Second as a decimal number [00,61].
+                %z  Time zone offset from UTC.
+                %a  Locale's abbreviated weekday name.
+                %A  Locale's full weekday name.
+                %b  Locale's abbreviated month name.
+                %B  Locale's full month name.
+                %c  Locale's appropriate date and time representation.
+                %I  Hour (12-hour clock) as a decimal number [01,12].
+                %p  Locale's equivalent of either AM or PM.
```

Comment:
Copied from `time.strftime()`, I see. This is very nice!

Perhaps here we should omit the codes referring to hours, minutes and seconds?

Also, where would one find more detailed info and the full list of format codes available? Perhaps consider a reference as to where more detailed information may be found, though I'm not sure what that should be. For `time.strftime()` we have: "Other codes may be available on your platform. See documentation for the C library strftime function."